src/App.tsx

```diff
@@ -15,48 +18,113 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabetical = 'Alphabetical',
+  Length = 'Length',
+}
+
+const sortGoods = (goodsArray: string[], sortType: SortType,
+  isReverse: boolean): string[] => {
+  let sortedGoods = [...goodsArray];
+
+  switch (sortType) {
+    case SortType.Alphabetical:
+      sortedGoods = sortedGoods.sort();
+      if (isReverse) {
+        sortedGoods.reverse();
+      }
```

Comment:
this code should not be duplicated in every case, is should be executed only once on after the switch block 
```
if (isReverse) {
     sortedGoods.reverse();
 }
```